mfa/callbacks/about.md

```diff
@@ -0,0 +1,32 @@
+# Two-Factor Authentication Callbacks
+Two-Factor Authentication (2FA) Callbacks (also known as [webhooks](https://webhooks.pbworks.com/w/page/13385124/FrontPage)) rely on DLRs (Delivery Receipts) sent to the Two-Factor Authentication (2FA) platform by both Bandwidth's Domestic and International platform. Since these two platforms send different DLR formats, the 2FA platform standardizes them into a singular format for simplicity and ease of consumption. The 2FA platform then takes these standardized DLRs and sends them via an HTTP POST to the specified URL that was set in the `callbackUrl` field in the [initial request](../methods/code/messaging.md). The `callbackUrl` field can be any publicly addressable url.
+
+## Message Callbacks Concepts
+- Both Domestic and International callbacks have the same fields and have the same format. However, some of these fields differ in values to denote between the two systems.
+- Domestic callbacks are retried for 24 hours if the 2FA platform is unable to deliver it to you.
+- International callbacks are not retried if the 2FA platform is unable to deliver it to you.
```

Comment:
Should we add `after the initial delivery attempt` to this sentence to be clear (assuming that's truly the case)

Reply:
updated